lib/Elevate/Components/EA4.pm

```diff
@@ -149,4 +150,13 @@ sub _restore_config_files ($self) {
     return;
 }
 
+sub _restore_imunify_phps ($self) {
+
+    my $php_pkgs = Elevate::StageFile::read_stage_file('ea4_imunify_packages');
+    return unless scalar $php_pkgs->@*;
+
+    $self->dnf->install(@$php_pkgs);
```

Comment:
so we're using yum after we restore EA? this means EA4 restore will fail and we're ok with that?

Reply:
The EA4 restore will be successful, but it will not install these packages since the backup profile considers them to be dropped packages.

Doing it this way came down to a design decision:

1.  Manually manipulate the profile file that `ea_current_to_profile` generates so that the imunify packages get reinstalled

or

2.  Track the "dropped" packages from Imunify 360 that EA4 doesn't know about, then install them after the EA4 restore completes

I went with option 2 here because I felt that it was safer and easier to proceed this way than with option 1.